src/app/consultancy/[freelancer_id]/page.tsx

```diff
@@ -44,6 +40,8 @@ import { axiosInstance } from '@/lib/axiosinstance';
 import { ProjectCard } from '@/components/cards/projectCard';
 import { Separator } from '@/components/ui/separator';
 import { ProjectStatus } from '@/utils/freelancer/enum';
+import { Input } from '@/components/ui/input';
```

Comment:
i think this wont be needed here

Reply:
Input is used in this page in formfield.